openff/utilities/provenance.py

```diff
@@ -59,4 +61,18 @@ def get_ambertools_version() -> Optional[str]:
             still returns `None`, but without a warning associated with the above failure.
     """
 
-    return _get_conda_list_package_versions().get("ambertools", None)
+    try:
+        return _get_conda_list_package_versions().get("ambertools", None)
+    except (
+        ValueError,  # Issue 98
+        subprocess.CalledProcessError,  # Issue 101
```

Comment:
Hard to guess which errors could bubble up, I don't want to be too broad but I also want to gobble up the few that I've seen